winml/test/api/LearningModelBindingAPITest.cpp

```diff
@@ -289,7 +296,7 @@ TEST_F(LearningModelBindingAPITest, ZipMapString)
 
 TEST_F(LearningModelBindingAPITestGpu, GpuSqueezeNet)
 {
-
+    GPUTEST
```

Comment:
>GPUTEST [](start = 4, length = 7)

same here. isn't the point of inheriting from the LearningModelBindingAPITestGpu class that you don't have to put this here? maybe I'm not understanding how that works